onnxruntime/core/framework/library_handles.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <utility>
+#include "core/platform/env.h"
+#include "core/common/path_string.h"
+#include "core/common/inlined_containers.h"
```

Comment:
>s.h" [](http://example.com/codeflow?start=39&length=4)

`_fwd` would be sufficient in this case and may help avoid issues with CUDA compiles when and if it gets indirectly included into CUDA (wierd things happen).

Reply:
Updated to use the _fwd version